ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -5,6 +5,8 @@ let prototyped_of_class = function
 let prototyped_of_field = function
   | "Repository", "gpgkey_path" ->
       Some "22.12.0"
+  | "pool", "uefi_certificates" ->
+      Some "22.18.0-next"
```

Comment:
This field hasn't been defined quite right in the datamodel... It is not a new field, but it has changed recently. Please remove this diff and I'll fix it separately.